pkg/oci/config.go

```diff
@@ -29,13 +29,13 @@ type AuthConfig struct {
 	UseInstancePrincipals bool   `yaml:"useInstancePrincipals"`
 	Region                string `yaml:"region"`
 	TenancyID             string `yaml:"tenancy"`
-	// CompartmentID is DEPRECIATED and should be set on the top level Config
+	// CompartmentID is DEPRECATED and should be set on the top level Config
 	// struct.
 	CompartmentID string `yaml:"compartment"`
 	UserID        string `yaml:"user"`
 	PrivateKey    string `yaml:"key"`
 	Fingerprint   string `yaml:"fingerprint"`
-	// PrivateKeyPassphrase is DEPRECIATED in favour of Passphrase.
+	// PrivateKeyPassphrase is DEPRECATED in favour of Passphrase.
```

Comment:
These should probably be in a separate PR don't you think as they're a separate concern? Deprecating compartment id has impacts on load balancers and other instance methods.